cudamapper/src/overlapper_minimap.cu

```diff
@@ -839,7 +950,45 @@ void OverlapperMinimap::get_overlaps(std::vector<Overlap>& fused_overlaps,
                                      int64_t min_bases_per_residue,
                                      float min_overlap_fraction)
 {
-    const int32_t n_anchors = d_anchors.size();
+#ifdef RUN_CPU
+    const size_t n_anchors = d_anchors.size();
+    std::vector<int32_t> predecessors(n_anchors, -1);
+    std::vector<double> scores(n_anchors, 0);
+    std::vector<Anchor> anchors(n_anchors);
+    std::vector<bool> select_mask(n_anchors, true);
+    std::vector<Overlap> overlaps_source(n_anchors);
+    cudautils::device_copy_n(d_anchors.data(), n_anchors, anchors.data(), _cuda_stream);
+    GW_CU_CHECK_ERR(cudaStreamSynchronize(_cuda_stream));
+
+    chain_anchors_in_block_cpu(anchors,
+                                scores,
+                                predecessors,
+                                n_anchors);
+
+    chainerutils::backtrace_anchors_to_overlaps_cpu(anchors,
+                                 //fused_overlaps,
+                                       overlaps_source,
+                                       scores,
+                                       select_mask,
+                                       predecessors,
+                                       n_anchors,
+                                       10); // min_score
```

Comment:
Changed min score hoping that we could pick up on the smaller values if there were any